core/deployment/src/main/java/io/quarkus/deployment/builditem/nativeimage/ExcludeConfigBuildItem.java

```diff
@@ -0,0 +1,35 @@
+package io.quarkus.deployment.builditem.nativeimage;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+/**
+ * A build item that allows extension to configure the native-image compiler to effectively
+ * ignore certain configuration files in specific jars.
+ *
+ * The {@code jarFile} property specifies the name of the jar file or a regular expression that can be used to
```

Comment:
No need to remove this ability, but I hope we'll never need regexes to match them :)

While we need this safeguard, we should not encourage using it if not as last resort.